terraform/main.tf

```diff
@@ -396,6 +396,10 @@ output "specific_manifests" {
   }
 }
 
+output "own_manifest_url" {
```

Comment:
nit: `own_manifest_base_url`, to connote that this isn't the full URL to a manifest but rather the URL relative to which manifests may be found.